src/example-types/groupedMetric.js

```diff
@@ -1,35 +1,13 @@
 let _ = require('lodash/fp')
 
-// Org -> Month -> $
-let example = {
-  metric: {
-    type: 'sum',
-    field: 'PO.IssuedAmount'
-  },
-  aggs: [
-    {
-      type: 'terms',
-      field: 'Organization.NameState.untouched',
-      data: {}
-    },
-    {
-      type: 'date_histogram',
-      field: 'PO.IssuedDate',
-      data: {
-        interval: 'month'
-      }
-    }
-  ]
-}
-
```

Comment:
We should comment this and not just delete them - and then move into the docs when we add documentation for this type.